docs/changes.md

```diff
@@ -1,6 +1,6 @@
 ## Changes
 
-
+- [`5.4`](https://github.com/ibm-mas/cli/releases/tag/5.4.0) Add new Manage bundle server config for SNO
```

Comment:
It would still be 5.3 I believe as 5.2 is the lastest at the moment

Reply:
fixed